addons/base_partner_address/models/migration.py

```diff
@@ -0,0 +1,171 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+# Odoo, an open source suite of business apps
+# This module copyright (C) 2015 bloopark systems (<http://bloopark.de>).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from openerp import api, models
+
+
+class BasePartnerAddressMigration(models.TransientModel):
+    _name = 'base.partner.address.migration'
+
+    @api.model
+    def migrate_contact_information(self):
```

Comment:
@kaerdsar Maybe it is good to add a short comment here why we use PL/PgSQL instead of cr.execute(sql_query) or ORM methods.